dashboard.py

```diff
@@ -108,11 +123,12 @@ def model_predictions_review():
         if not os.path.exists(model_dir):
             os.mkdir(model_dir)
 
-        name = filename.split("/")[0] + "-" + str(idx + start_index) + "-"
-        pred_piece.source["midi_filename"] = train_cfg.run_name + "/" + name + os.path.basename(filename)
+        name = f"{filename.split('.')[0].replace('/', '-')}-{idx + start_index}-"
+        directory = "tmp/dashboard/"
+        pred_piece.source["midi_filename"] = model_dir + "/" + name + ".mid"
 
-        name = filename.split("/")[0] + "-" + str(idx) + "-qv-" + bins + "-"
-        quantized_vel_piece.source["midi_filename"] = "common/" + name + os.path.basename(filename)
+        name = f"{filename.split('.')[0].replace('/', '-')}-{idx + start_index}-qv-{bins}-{dataset.sequence_len}-"
+        quantized_vel_piece.source["midi_filename"] = directory + "common/" + name + ".mid"
```

Comment:
In the maestro datasets we have the information about composer name and the piece title - we should make effort to display it here.

The approach I think should work for this, is to track additional field for every record with a dumped source information:

```python
record = {
    ...
    'source': json.dumps(piece.source)
}
```

This should be enough to deal with the fact, that this `source` is not unified between datasets. 

You should also be able to track the actual start and finish note indexes and display those on the dashboard as well.